fubectl.source

```diff
@@ -12,6 +12,9 @@ _isClusterSpaceObject() {
 # [k] like g for git but 233% as effective!
 alias k="kubectl"
 
+# [kw] kw get po,svc,md
+alias kw="watch kubectl"
```

Comment:
```suggestion
alias kw="watch kubectl get"
```

Reply:
i also use this for top: `kw top node` and sometimes `kw describe pod/askoijhdashasdljhikosd`

would it make sense to create specific shortcuts like this?

* kwg -> `watch kubectl get`
* kwd -> `watch kubectl describe`
* kwt -> `watch kubectl top`
* kwdiff -> `watch kubectl diff`

Reply:
🤷‍♂ I had never used watch with describe or top. my local alias kw will stay at get operation anyway :)

Reply:
Agreed, that watching describe and diff is weird. For top I can see it. I would still recommened some proper monitoring, but hey. :smile:

Reply:
ok i see your points :) maybe we can alias `kw` and/or `kwg` to `watch kubectl get` and `kwt` to `watch kubectl top`?